src/interactors/interactors.test.js

```diff
@@ -70,4 +77,18 @@ describe('Interactors', () => {
     await updateSchema.execute(mockThing);
     expect(cloudMocks.mockUpdateSchema).toHaveBeenCalled();
   });
+
+  test("publishData: should execute interactor correctly when publish thing's data on cloud", async () => {
```

Comment:
Same on `gerund` :point_down: 
```suggestion
  test("publishData: should execute interactor correctly when publishing a thing's data to the cloud", async () => {
```

Reply:
Done!